dart/lib/src/sentry_client.dart

```diff
@@ -485,6 +485,75 @@ class SentryClient {
     );
   }
 
+  @internal
+  Future<void> captureLog(
```

Comment:
Matched attributes and traceId application as close as i could to the [Python implementation](https://github.com/getsentry/sentry-python/blob/b16fa5ffbad39843ebd2e9bc4ea6e91c0c9aa192/sentry_sdk/client.py#L213).

Reply:
we can also leave parameterize strings out for now